### PR TITLE
Mateo pull

### PR DESCRIPTION
**Pull request by mjignacio**

### CODE REVIEW

File: scripts/collect_images.py [hunk 1]
```diff
@@ -7,11 +7,12 @@
 
 
 INPUT = 'data/Q1ProjectStructureList.xlsx'
-OUTPATH = 'data/images/'
+OUTPATH = '../data/images/'
```
Comment (adenny640) [inline scripts/collect_images.py]: outpath shouldn't be modified as script should be run from root street watch directory

File: scripts/collect_images/mateo.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    ["32.75399,-117.132", '600x400',0,190,60],
+    ["32.75399,-117.132", '600x400',0,190,100],
+    ["32.75399,-117.132", '600x400',0,190,120],
+    ["32.75381,-117.132", '600x400',0, 270,100],
+    ["32.75381,-117.132", '600x400',0, 270,60],
+    ["32.75381,-117.132", '600x400',0, 270,120],
+    ["32.75377,-117.132", '600x400',0,350,100],
+    ["32.75377,-117.132", '600x400',0,350,120],
+    ["32.75377,-117.132", '600x400',0,350,60],
+    ["32.75371,-117.132",'600x400',0,100],
```
Comment (adenny640) [inline scripts/collect_images/mateo.py]: Missing parameters from 217-219 breaks code

File: scripts/collect_images/mateo.py [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+
+    structures = [first,second,third,fourth,fifth,sixth,seventh,eigth,ninth,tenth]
+
+    for structure in structures:
+        for parama in structure:
+            capture_street_view_image(parama[0], parama[1], parama[2], parama[3], parama[4], outpath)
+       
+
+if __name__== "__main__":
+    generate_images(outpath='../data/images/')
```
Comment (adenny640) [inline scripts/collect_images/mateo.py]: Same thing about running from root directory, should just be 'data/images'

**Review by adenny640 — COMMENTED** [review]
Script works and generates images, code breaks on structure 7 due to missing parameters. Would also make it so that the images don't open as they are being downloaded.

File: scripts/collect_images/mateo.py [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+    ["32.8699,-117.204", '600x400',0, 35,40],
+    ["32.8699,-117.204", '600x400',0, 35,20]]
+
+
+    # Fifth
+    fifth = [["32.8704883,-117.206585", '600x400',0, 80, 30],
+    ["32.8704883,-117.206585", '600x400',0, 80, 10],
+    ["32.8704883,-117.206585", '600x400',0, 80, 20],
+    ["32.8704883,-117.206085", '600x400', 0, 120, 60],
+    ["[32.8704883,-117.206085", '600x400', 0, 120, 45],
```
Comment (pdashk) [inline scripts/collect_images/mateo.py:175]: minor bug here

**Review by pdashk — APPROVED** [review]
see minor bug

**Pull request merged**